client/package.json

```diff
@@ -18,8 +18,8 @@
     "typings": "./lib/index",
     "dependencies": {
         "glob-to-regexp": "^0.3.0",
-        "vscode-jsonrpc": "^5.0.0",
-        "vscode-languageclient": "^6.0.0",
+        "vscode-jsonrpc": "^5.0.1",
+        "vscode-languageclient": "^6.1.3",
```

Comment:
I see [here](https://github.com/microsoft/vscode/blob/94cb3bc814d6664bc701fd9d074d5260cf55505f/extensions/css-language-features/package.json#L809) and [here](https://github.com/microsoft/vscode/blob/94cb3bc814d6664bc701fd9d074d5260cf55505f/extensions/html-language-features/package.json#L204) that `6.1.0` version of `vscode-languageclient` is used for `0.20.x` version of VS Code.
I don't know if it's critical, but maybe it makes sense to keep this version close to one which is used in VS Code.